### DISCUSSION TITLE
How to run custom authorization code only for specific routes?

### DISCUSSION THREAD

**Discussion by fauresco** [ThreeMammals/Ocelot#1737] [Q&A]
I have written a custom authorization routine hooked up in Ocelot `PreAuthorizationMiddleware`. 

How can I run this just for specific routes?

Is there any concept like adding a custom property to a route? Something like `UseAdvancedAuthorization=true` and somehow access this in the `PreAuthorizationMiddleware` to decide if it will handle the authorization or just let the default `AuthorizationMiddleware` do its job.

**Comment by ahmedanwar100** [comment]
@fauresco, have you read this doc https://ocelot.readthedocs.io/en/latest/features/authentication.html. May be it can help
